### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- update examples

I was testing the example and that was missing a few things, also did a cleanup and update in the same

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,17 +22,17 @@ Example using a pinned version:
...
 
-    name: Install Cosign and test presence in path
+    name: Install Cosign
     steps:
       - name: Install Cosign
-        uses: sigstore/cosign-installer@main
+        uses: sigstore/cosign-installer@v3.0.5
         with:
-          cosign-release: 'v2.0.2'
+          cosign-release: 'v2.0.0'
```
Comment (bobcallaway) [inline README.md:35]: We should recommend 2.0.2 as it contains the TUF root fetched over cdn

Comment (cpanato) [reply README.md:35]: ok, that was just an example, but will update

Comment (cpanato) [reply README.md]: done, PTAL

Comment (bobcallaway) [reply README.md]: Understood, but since people will copy and paste this id rather not recommend an old version

**Pull request merged**